sdktests/common_tests_poll_request.go

```diff
@@ -33,6 +36,53 @@ func (c CommonPollingTests) RequestMethodAndHeaders(t *ldtest.T, credential stri
 	})
 }
 
+func (c CommonPollingTests) LargePayloads(t *ldtest.T) {
+	flags := make([]ldmodel.FeatureFlag, 100)
```

Comment:
I'd be curious to know the size of this in bytes. Like - is this 1 kb? 10kb? Should we be using 1000 flags?

Reply:
100 flags was ~59k. Jumped up to 1,000 just to be safe. Now we are around 585k.